org.sugarlabs.FractionBounce.json

```diff
@@ -0,0 +1,235 @@
+                "pip3 install --no-index --find-links \"file://${PWD}\" --install-option=\"--prefix=${FLATPAK_DEST}\" six"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/dd/bf/4138e7bfb757de47d1f4b6994648ec67a51efe58fa907c1e11e350cddfca/six-1.12.0.tar.gz",
+                    "sha256": "d16a0141ec1a18405cd4ce8b4613101da75da0e9a7aec5bdd4fa804d0e0eba73"
+                }
+            ]
+        },
+        {
+            "name": "telepathy-glib",
+            "config-opts": ["--disable-static", "--disable-gtk-doc"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://www.github.com/TelepathyIM/telepathy-glib.git",
+                    "tag": "telepathy-glib-0.24.1",
+                    "commit": "c834ef1a50a97cbfae7a0d560ff806fd5aa7ffe8"
+                },
+                {
+                    "type": "patch",
+                    "path": "telepathy-glib-uniquify.patch"
+                }
+            ]
+        },
+        {
+            "name": "sugar-toolkit-gtk3",
+            "no-parallel-make": true,
+            "config-opts": [
+                "--with-python3"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.sugarlabs.org/sources/sucrose/glucose/sugar-toolkit-gtk3/sugar-toolkit-gtk3-0.116.tar.xz",
+                    "sha256": "745a360577efceb10f57ca4cbad5f9ec61f7ed6cc381eeb72422bbd7455b770b"
+                }
+            ]
+        },
+        {
+            "name": "sugarapp",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links ${PWD} --install-option=\"--prefix=${FLATPAK_DEST}\" $PWD"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/tchx84/sugarapp.git",
+                    "tag": "v1.11",
+                    "commit": "68b300b93be31dca3e6d621c107c703bf140aa03"
+                }
+            ]
+        },
+        {
+            "name": "fractionbounce",
+            "buildsystem" : "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST} --skip-install-desktop-file --skip-install-mime"
+            ],
+            "sources" : [
+                {
+                    "type": "git",
+                    "url": "https://github.com/sugarlabs/fractionbounce",
+                    "commit": "4c6d978ca340b036318ac6e714d7cbf1f568d4a6"
+                },
+                {
+                    "type": "patch",
+                    "path": "fractionbounce-port.patch"
+                },
+                {
+                "type": "patch",
+                "path": "fractionbounce-info.patch" 
+                }
```

Comment:
Don't forget the "-monitors.patch". 

https://github.com/tchx84/sugarapp/blob/master/flatpak-guide-common-issues.md

Just run `grep "Gdk.Screen" -r .` and you will find what needs to be changed, The only exception is `Gdk.Screen.get_default().connect('size-changed', self._configure_cb)`, because is still useful to detect overall changes.

Reply:
Fixed